backend/canvas_app_explorer/canvas_lti_manager.py

```diff
@@ -0,0 +1,21 @@
+from typing import List
+from canvasapi import Canvas
+from .data_class import ExternalTool
+
+class CanvasLtiManager:
+    def __init__(self, API_URL: str, API_KEY: str, course_id: int):
```

Comment:
These parameters should be lowercase. There will be some other changes if these changed.

```suggestion
    def __init__(self, api_url: str, api_key: str, course_id: int):
```